health-sdk/sdk/src/doc/source/flow.rst

```diff
@@ -85,6 +85,51 @@ into one final document:
     }
 
 Check which documents/invoices are payable
+---------------------------------
+
+``GiniHealthSDK`` provides a method to delete a payment request. You can do this by calling ``giniHealth.deletePaymentRequest(...)`` with a payment request ID.
+
+.. code-block:: kotlin
+
+    coroutineScope.launch {
+        // Delete  payment requests
+        val deletePaymentRequest = giniHealth.deletePaymentRequest(paymentRequestId)
+
+        when (deletePaymentRequest) {
+            is Resource.Success -> {
+                // `null` will be returned here
+            }
+            is Resource.Error -> // Handle Error
+            is Resource.Cancelled -> //  Handle cancellation
+        }
+    }
+
+
+To delete a payment request, you must first create one to obtain a valid payment Request Id. Ensure that the request has not expired before attempting deletion.
+---------------------------------
+
+``GiniHealthSDK`` provides a  method to delete multiple payment request at once. You can do this by calling ``giniHealth.deletePaymentRequests(...)`` with a list of payment request IDs. The call will only succeed if all payment request were successfully deleted. If any payment request is invalid, unauthorized, or not found, the entire deletion request will fail, and no payment requests will be deleted. In the case of failures, an error or type ``DeletePaymentRequestErrorResponse`` will be provided, with more insight into why the deletion failed.
+
+.. code-block:: kotlin
+
+    // Assuming `paymentRequestIds` is a list of `String` which
+    // representing the IDs of the payment requests to be deleted
+
+    coroutineScope.launch {
+        // Delete multiple payment requests at once
+        val deletePaymentRequests = giniHealth.deletePaymentRequests(paymentRequestIds)
+
+        when (deletePaymentRequests) {
+            is Resource.Success -> {
+                // `null` will be returned here
+            }
+            is Resource.Error -> // Handle `DeletePaymentRequestErrorResponse`
+            is Resource.Cancelled -> // Handle `DeletePaymentRequestErrorResponse`
+        }
+    }
+
+
+To check which payment requests are deletable, ensure that they are valid, authorized, and have not expired before attempting deletion.
```

Comment:
[Clarification] Is this necessary? This would imply that the client needs to fetch the payment request to check it before trying to delete it. I think that's why we have the different errors that the BE returns, so the client doesn't need to check beforehand. 
Also, not sure how one would check if a payment request is authorised 🤔